### DISCUSSION TITLE
If KeyBinding of the Printscreen key with ScreenGrab program possible?

### DISCUSSION THREAD

**Discussion by rajibando** [lxqt/lxqt#2607] [Q&A]
I was suggested by the earlier maintainer, Mr. Artem Galichkin, to get in touch with lxqt team, as "... Version 2.0 is developed, supported and maintained by the lxqt project team ..."

ScreenGrab program is included in the debian-live-1 2.6.0-amd64-lxqt.iso and installs by default.

Could the keyboard key PrintScreen be bound, as stated? That is, effectively, if ScreenGrab could be run as a daemon upon start-up? Could guidance be provided in this regard please?

**Comment by isf63** [comment]
ScreenGrab, or equivalently, `lximage-qt -s`, doesn't need to be run as a daemon. When bound to a key the program is freshly started.

To add a keybinding under X11, go to Preferences > Shortcut Keys and add an entry of type Command that runs either `screengrab` or `lximage-qt -s`.

For Wayland compositors, it's more complex and you need to edit the specific compositor's config files to add a keybinding. In principle it's the same - adding a 'command' keybinding, but there is no unified way to do it on Wayland currently.

**Reply by rajibando** [reply]
Apologies for the delay.

The PrintScreen key was set up for the purpose with:
Preferences > LXQt Settings > Shortcut Keys (Global Actions Manager)

When reboot didn't enable the desired action, all the key+combinations were tried.
Strangely, none works as mentioned.
